src/features/applications/components/application-method-definitions.tsx

```diff
@@ -1,5 +1,7 @@
-import algosdk from 'algosdk'
+import { ABITransactionType } from '@algorandfoundation/algokit-utils/abi'
+import { TransactionType as UtilsTransactionType } from '@algorandfoundation/algokit-utils/transact'
```

Comment:
```suggestion
import { TransactionType } from '@algorandfoundation/algokit-utils/transact'
```

Probably no need for this alias.

Reply:
We will need this, there is another `TransactionType` import below
```
import { AppCallTransaction, TransactionType } from '@/features/transactions/models'
```
I was considering to remove the lora internal `TransactionType` but decided not to because the lora version has user friendly names, for example, `AssetTransfer = 'Asset Transfer'`